### DISCUSSION TITLE
新组件-浮动面板

### DISCUSSION THREAD

**Discussion by RYGRIT** [Moonofweisheng/wot-design-uni#597] [Show and tell]
目前已经完成的效果

![floatPanel](https://github.com/user-attachments/assets/ee25a6b0-23c6-453e-a87e-3d8d283d9744)

大家有什么需求或者更好的点子吗？

**Comment by Moonofweisheng** [comment]
很棒

**Reply by RYGRIT** [reply]
现在内容和头部均可以拖拽，想支持仅头部拖拽，目前有一种方案，就是设置一个 data-* 属性，来区分它是头部还是内容。你有更好的方案吗？

**Reply by Moonofweisheng** [reply]
> 现在内容和头部均可以拖拽，想支持仅头部拖拽，目前有一种方案，就是设置一个 data-* 属性，来区分它是头部还是内容。你有更好的方案吗？

可以先提交一个pr上来，我们一起交流下。

**Reply by RYGRIT** [reply]
👌

**Comment by elarbor** [comment]
能不能加个【时间轴】的组件啊

**Reply by RYGRIT** [reply]
@Moonofweisheng 采纳吗

**Comment by Moonofweisheng** [comment]
嗯，我觉得没问题👍



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年9月30日(周一) 上午8:38
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Moonofweisheng/wot-design-uni] 新组件-浮动面板 (Discussion #597)




 
@Moonofweisheng 采纳吗

**Reply by RYGRIT** [reply]
👌 那我尝试一下

@chergn 你是否有关于时间轴组件的出色想法呢？又或者你是否可以提供相关的效果图呢？

**Comment by qzsiniong** [comment]
希望可以增加title
![IMG_4822](https://github.com/user-attachments/assets/56bf2a9c-899b-494a-a1df-18b44a1f9264)
